pages/common/zapier-push.md

```diff
@@ -0,0 +1,16 @@
+# zapier push
+
+> Build and upload a Zapier integration.
+> More information: <https://github.com/zapier/zapier-platform/blob/main/packages/cli/docs/cli.md#push>.
```

Comment:
```suggestion
> More information: <https://platform.zapier.com/reference/cli#push>.
```

Reply:
sounds good to me, ill probably make one as soon as this last one is merged